tests/integration/nm/ipv4_test.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate import nm
+
+from .testlib import mainloop
+
+
+ETH1 = 'eth1'
```

Comment:
IMHO it is actually nicer to give the constant a describing name, such as `TEST_INTERFACE` to simplify changing this to a different name without having to rename the constant everywhere as well since using `ETH1 = 'testveth'` would not be that nice.

Reply:
Done